src/apic-extension/azext_apic_extension/aaz/latest/apic/service/_import_from_apim.py

```diff
@@ -22,9 +22,9 @@ class ImportFromApim(AAZCommand):
     """
 
     _aaz_info = {
-        "version": "2024-03-01",
+        "version": "2024-03-15-preview",
```

Comment:
@arpishahmsft the rest api spec 2024-03-01 does not contain import from apim definition, should I use `2024-03-15-preview` here, or manually change back to `2024-03-01`?

Reply:
@arpishahmsft Manually changed to `2024-03-01` per your suggestion